scipy/stats/_hypotests.py

```diff
@@ -312,7 +316,7 @@ def cramervonmises(rvs, cdf, args=()):
     ----------
     .. [1] Cramér-von Mises criterion, Wikipedia,
            https://en.wikipedia.org/wiki/Cram%C3%A9r%E2%80%93von_Mises_criterion
-    .. [2] Csorgo, S. and Faraway, J. (1996). The Exact and Asymptotic
+    .. [2] Csörgő, S. and Faraway, J. (1996). The Exact and Asymptotic
```

Comment:
CI failures are related to special unicode characters I think:

`line 319, position 17: character 'ő', code point U+0151`

I believe we have this guard in place because the introduction of some unicode to files can be problematic.